packages/web-app-files/src/components/Search/List.vue

```diff
@@ -2,6 +2,28 @@
   <div class="files-search-result oc-flex">
     <files-view-wrapper>
       <app-bar :has-bulk-actions="false" :side-bar-open="sideBarOpen" />
+      <div class="files-search-result-filter oc-mx-m oc-mb-m">
+        <item-filter
+          v-if="availableTags.length"
+          ref="tagFilter"
+          :allow-multiple="true"
+          :filter-label="$gettext('Tags')"
+          :filterable-attributes="['label']"
+          :items="availableTags"
+          :option-filter-label="$gettext('Filter tags')"
+          :show-option-filter="true"
+          class="files-search-filter-tags oc-mr-s"
```

Comment:
would prefer a data-testid over class since it seems its only used as a test selector

Reply:
Alright, as discussed in chat: Let's keep the class as we might want to style it later and there is no real need to handle this via `testid` because it doesn't have an id.